tests/protocol/mockwiredata.py

```diff
@@ -327,6 +325,12 @@ def mock_gen_trans_cert(self, trans_prv_file, trans_cert_file):
         with open(trans_cert_file, 'w+') as cert_file:
             cert_file.write(self.trans_cert)
 
+    @staticmethod
```

Comment:
when the status is PUT via the hostgaplugin, it is wrapped in an enclosing object as property "content" and base64-encoded.

this function unwraps it and decodes it